### DISCUSSION TITLE
JinxFramer

### DISCUSSION THREAD

**Discussion by kab2023** [vvip-68/LedPanelWiFi#8] [Вопросы и ответы]
Подскажите, пожалуйста, где скачать JinxFramer?

**Comment by vvip-68** [comment]
![image](https://github.com/vvip-68/LedPanelWiFi/assets/39353139/2d710881-0b24-4860-9811-12db627b9571)

**Comment by kab2023** [comment]
Спасибо

**Comment by kab2023** [comment]
Скачал, запускаю, пишет, что программа не может исполнятся на моем компьютере

**Reply by vvip-68** [reply]
Сочувствую...
